src/databricks/labs/ucx/config.py

```diff
@@ -94,8 +94,7 @@ class _Config(Generic[T]):
 
     @classmethod
     @abstractmethod
-    def from_dict(cls, raw: dict[str, Any]) -> T:
```

Comment:
These changes are irrelevant to this PR, revert them and create a separate PR if you want them in

Reply:
I re cloned and only committed changes to the query files relevant for this PR.